test/hotspot/jtreg/vmTestbase/metaspace/share/TriggerUnloadingWithWhiteBox.java

```diff
@@ -20,7 +20,7 @@
  * or visit www.oracle.com if you need additional information or have any
  * questions.
  */
-package vm.share.gc;
+package metaspace.share;
```

Comment:
There's a triggerUnloading call here:

test/lib/jdk/test/lib/classloader/ClassUnloadCommon.java

You might be able to also remove this file (and maybe the others) and use the ClassUnloadCommon version.

Reply:
Thanks. I filed 
https://bugs.openjdk.org/browse/JDK-8334320
The functionality is little different so more testing might be required for changed tests.